app/src/main/java/agency/five/codebase/android/movieapp/ui/home/mapper/HomeScreenMapperImpl.kt

```diff
@@ -0,0 +1,46 @@
+package agency.five.codebase.android.movieapp.ui.home.mapper
+
+import agency.five.codebase.android.movieapp.R
+import agency.five.codebase.android.movieapp.model.Movie
+import agency.five.codebase.android.movieapp.model.MovieCategory
+import agency.five.codebase.android.movieapp.ui.component.MovieCategoryLabelTextViewState
+import agency.five.codebase.android.movieapp.ui.component.MovieCategoryLabelViewState
+import agency.five.codebase.android.movieapp.ui.home.HomeMovieCategoryViewState
+import agency.five.codebase.android.movieapp.ui.home.HomeMovieViewState
+
+class HomeScreenMapperImpl : HomeScreenMapper {
+    override fun toHomeMovieCategoryViewState(
+        movieCategories: List<MovieCategory>,
+        selectedMovieCategory: MovieCategory,
+        movies: List<Movie>
+    ): HomeMovieCategoryViewState {
+        return HomeMovieCategoryViewState(movieCategories.map { category ->
```

Comment:
this could be an expression function (check this https://stackoverflow.com/questions/64856717/kotlin-expression-fun-vs-normal-fun-differences)

also, the formatting is not okay. movieCategories.map should go in a new line, and when you have multiple parameters please add parameter names